ext/standard/tests/streams/proc_open_bug69900.phpt

```diff
@@ -48,8 +48,8 @@ fclose($pipes[1]);
 proc_close($process);
 
 /* It is expected that the first call takes more than the limit.
- * Allow one more to account for a possible process switch. */
-if ($moreThanLimit > 2) {
+ * Allow two more to account for possible process switch, etc. */
+if ($moreThanLimit > 3) {
     echo "fgets() took more than $max_ms ms $moreThanLimit times\n";
 }
```

Comment:
Do you remember which builder this one failed on?

Reply:
I've seen this one fail repeatedly. At least once on Travis CI, in the S390X environment, but I think I have seen it in other environments as well.